### DISCUSSION TITLE
Show only a half of the visualizer

### DISCUSSION THREAD

**Discussion by SalahAdDin** [katspaugh/wavesurfer.js#3872] [Ideas]
We have the following design for our media player:
![image](https://github.com/user-attachments/assets/9c8cb81a-8713-4ff6-8f97-947b8ab664b6)

We could not find any documentation about how to show only half of the visualization.

It would be nice to have any property or a way to style the visualizer in this way.

**Comment by katspaugh** [comment]
You can do it with `barAlign: 'bottom'`.
https://wavesurfer.xyz/docs/types/wavesurfer.WaveSurferOptions
https://wavesurfer.xyz/examples/?all-options.js

**Reply by SalahAdDin** [reply]
> You can do it with `barAlign: 'bottom'`. https://wavesurfer.xyz/docs/types/wavesurfer.WaveSurferOptions https://wavesurfer.xyz/examples/?all-options.js

Man, I wish I had made this comment months ago when I needed it!

Thank you very much!
